test/e2e/specs/blocks/shared/block-smoke-testing.ts

```diff
@@ -68,7 +68,10 @@ export function createBlockTests( specName: string, blockFlows: BlockFlow[] ): v
 					const blockHandle = await editorPage.addBlockFromSidebar(
 						blockFlow.blockSidebarName,
 						blockFlow.blockEditorSelector,
-						{ noSearch: true, blockFallBackName: blockFlow.blockTestFallBackName }
+						{
+							noSearch: blockFlow.noSearch === false ? false : true,
```

Comment:
Nitpicky - this works well enough, but I would use null coalescing here:  
```suggestion
							noSearch: blockFlow.noSearch ?? true,
```
Then, if there's any non-nullish value in `noSearch` (anything other than `null` or `undefined`), it'd be used. Otherwise, it'd default to `true`. But that's just a preference - explicitly checking for `false` also works.